enumerables.rb

```diff
@@ -0,0 +1,332 @@
+# rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLength, Metrics/PerceivedComplexity, Lint/Void
+p '-----------|||||--------'
+p '----------{(o o)}--------'
+p '--------ooO-(_)-Ooo------'
+# Initial comment
+module Enumerable
+  def my_each
```

Comment:
My_each when a block is given and when it is an array, should return the array itself after calling the given block once for each element in the array.

![Screen Shot 2020-07-21 at 7 16 41 PM](https://user-images.githubusercontent.com/52430951/88119666-c470bb80-cb86-11ea-9dc9-eb1b28657536.png)